src/scripts/main.js

```diff
@@ -1 +1,12 @@
 'use strict';
+
+const largeImg = document.getElementById('largeImg');
+const thumbs = document.getElementById('thumbs');
+
+thumbs.addEventListener('click', (e) => {
+  e.preventDefault();
+
+  const img = e.target.closest('.list-item__link');
```

Comment:
Add a condition to check if the closest element with the class '.list-item__link' is found before assigning its href to largeImg.src